doc/tutorials/platformer/app/lib/extensions/random.dart

```diff
@@ -0,0 +1,8 @@
+import 'dart:math';
+
+Random random = Random();
+
+extension RandomExtension on Random {
+  double fromRange(double min, double max) =>
+      (nextDouble() * (max - min + 1)).floor() + min;
```

Comment:
If you need a random double in the range [min, max], then there shouldn't be +1 or floor().

Reply:
I got it from some SO answer.  I didn't really look at it to be honest, just snagged the extension and ran - it worked, so I didn't think about it.  I actually think this can be reduced a good bit as really, I am doing integers not doubles.